specs/binding-kafka-grpc.spec/src/main/scripts/io/aklivity/zilla/specs/binding/kafka/grpc/streams/kafka/bidi.stream.rpc/client.rpt

```diff
@@ -22,6 +22,7 @@ write zilla:begin.ext ${kafka:beginEx()
                               .merged()
                                   .capabilities("FETCH_ONLY")
                                   .topic("requests")
+                                  .groupId("zilla-kafka-grpc")
```

Comment:
This needs to be configurable, I'm thinking as part of `KafkaConfiguration`, agree?